src/models/User.ts

```diff
@@ -1,32 +1,73 @@
+import firebase from 'firebase';
 import {
-  Grade, Role, ScholarshipType, Gender,
+  Grade, ScholarshipType, Gender, ScholarshipAmount,
 } from './enums';
 
+export enum Role{
+  Reader,
+  Contributor,
+  Moderator,
+  Admin
+}
+export enum NotificationType{
+  None,
+  email,
+  mobile
+}
+
 interface UserProps {
-  age: number;
+  DOB: Date | undefined;
   schoolYear: Grade;
   role: Role;
   gender: Gender;
-  scholarshipPreferences: {
-    type: ScholarshipType,
-    major: string,
-    amount: number,
-    school: string,
-    schoolYear: Grade,
-    location: string
-  };
-  otherSettings: {
-    notifications: boolean,
-    deadlineNotifications: boolean,
-    relatedNotifications: boolean
+  scholarshipPrefs: ScholarshipPreferences;
+  otherSettings: NotificationSettings;
+}
+interface AuthProps {
+  displayName: string | null,
+  email: string | null,
+  emailVerified: boolean | null,
+  phoneNumber: string | null,
+  photoURL: string | null,
+  uid: string | null,
+}
+interface ScholarshipPreferences{
+  type: ScholarshipType;
+  major: string[] | undefined;
+  amount: {
+    type: ScholarshipAmount | undefined;
+    min: number | undefined;
+    max: number | undefined;
   }
+  school: string[];
+  schoolYear: Grade;
```

Comment:
My idea behind this was if a user was also interested in scholarships for next year. 
But I think it makes more sense using the filters for that.

Reply:
Ohh I see, this might be relevant then if they're considering multiple schools where `Grade` is different. Otherwise I would expect this to just be the current Grade+1